### ISSUE TITLE
Plugin wrongly detects hotwords inside GCODE comments

### ISSUE THREAD

**Issue by foosel** [OctoPrint/OctoPrint-FileCheck#1]
Example of a false positive:

```
; start_gcode = G28 ; home all axes\nG1 Z5 F5000 ; lift nozzle\nG1 X1 F{travel_speed} ; Lifts the nozzle at speed {travel_speed}
```

Plugin must only match things like `{travel_speed}` outside of comments.

See https://community.octoprint.org/t/disable-travel-speed-notice/22815/7?u=foosel

**Comment by thirstyice** [comment]
After updating, it now gives me a false positive on this file: 
[Filament switch holder.gcode.txt](https://github.com/OctoPrint/OctoPrint-FileCheck/files/5447153/Filament.switch.holder.gcode.txt)

Doing a ctrl-f, the only two instances of "travel" in the file are both in the comments at the end.

**Comment by foosel** [comment]
I cannot reproduce this against the current version (2020.8.7). The provided file uploads without any warnings.

Can you please share your `octoprint.log`?

**Comment by thirstyice** [comment]
I can't replicate it anymore either.
🤷
